Potential regression introduced on code or flaky tests

### Expected behavior

- The `Lisk Framework Integration` pipeline do not randomly fail

### Actual behavior

- Pipeline randomly fails

### Steps to reproduce

- Build the project on Jenkins
- Check logs in the comments

### Which version(s) does this affect? (Environment, OS, etc...)

development branch

`Lisk Framework Integration` random failure example: 

```
[2020-11-12T12:07:56.603Z] + npm run test:integration

[2020-11-12T12:07:56.603Z] 

[2020-11-12T12:07:56.603Z] > lisk-framework@0.7.0-alpha.3 test:integration /home/lisk/workspace/lisk-sdk_PR-5969/framework

[2020-11-12T12:07:56.603Z] > jest --config=./test/integration/jest.config.js

[2020-11-12T12:07:56.603Z] 

[2020-11-12T12:08:00.793Z] PASS test/integration/controller/ipc/ipc_server.spec.ts

[2020-11-12T12:08:00.793Z] PASS test/integration/controller/ipc_channel_without_bus.spec.ts

[2020-11-12T12:08:00.793Z] PASS test/integration/controller/in_memory_channel.spec.ts

[2020-11-12T12:08:01.053Z] PASS test/integration/controller/ipc_channel.spec.ts

[2020-11-12T12:08:01.622Z] PASS test/integration/node/forger/transaction_pool.spec.ts

[2020-11-12T12:08:01.881Z] PASS test/integration/node/processor/transaction_order.spec.ts

[2020-11-12T12:08:01.881Z] PASS test/integration/node/genesis_block.spec.ts

[2020-11-12T12:08:02.820Z] PASS test/integration/controller/ipc/ipc_client.spec.ts

[2020-11-12T12:08:03.755Z] PASS test/integration/node/processor/delete_block.spec.ts

[2020-11-12T12:08:13.726Z] FAIL test/integration/node/processor/process_block.spec.ts (13.135 s)

[2020-11-12T12:08:13.726Z]   ● Process block › given a block with invalid properties › when block has tie break BFT properties › should replace the last block

[2020-11-12T12:08:13.726Z] 

[2020-11-12T12:08:13.726Z]     Invalid block reward: 0 expected: 125000000

[2020-11-12T12:08:13.726Z] 

[2020-11-12T12:08:13.726Z]       100 | 		const expectedReward = isCompliant ? reward : reward / BigInt(4);

[2020-11-12T12:08:13.726Z]       101 | 		if (blockHeader.reward !== expectedReward) {

[2020-11-12T12:08:13.726Z]     > 102 | 			throw new Error(

[2020-11-12T12:08:13.726Z]           | 			      ^

[2020-11-12T12:08:13.726Z]       103 | 				`Invalid block reward: ${blockHeader.reward.toString()} expected: ${expectedReward.toString()}`,

[2020-11-12T12:08:13.726Z]       104 | 			);

[2020-11-12T12:08:13.726Z]       105 | 		}

[2020-11-12T12:08:13.726Z] 

[2020-11-12T12:08:13.726Z]       at BFT.verifyBlockHeader (../elements/lisk-bft/src/bft.ts:102:10)

[2020-11-12T12:08:13.726Z]       at Processor._processValidated (src/node/processor/processor.ts:347:3)

[2020-11-12T12:08:13.726Z]       at src/node/processor/processor.ts:214:6

[2020-11-12T12:08:13.726Z]       at Mutex.runExclusive (../elements/lisk-utils/src/job_handlers/mutex.ts:37:11)

[2020-11-12T12:08:13.726Z]       at Processor.process (src/node/processor/processor.ts:126:3)

[2020-11-12T12:08:13.726Z]       at Object.<anonymous> (test/integration/node/processor/process_block.spec.ts:367:5)

[2020-11-12T12:08:13.726Z] 

[2020-11-12T12:08:15.101Z] PASS test/integration/node/synchronizer/temp_block.spec.ts (15.688 s)

[2020-11-12T12:08:15.362Z] 

[2020-11-12T12:08:15.362Z] Test Suites: 1 failed, 10 passed, 11 total

[2020-11-12T12:08:15.362Z] Tests:       1 failed, 66 passed, 67 total

[2020-11-12T12:08:15.362Z] Snapshots:   1 passed, 1 total

[2020-11-12T12:08:15.362Z] Time:        17.245 s, estimated 23 s

[2020-11-12T12:08:15.362Z] Ran all test suites.

[2020-11-12T12:08:15.362Z] npm ERR! code ELIFECYCLE

[2020-11-12T12:08:15.362Z] npm ERR! errno 1

[2020-11-12T12:08:15.362Z] npm ERR! lisk-framework@0.7.0-alpha.3 test:integration: `jest --config=./test/integration/jest.config.js`

[2020-11-12T12:08:15.362Z] npm ERR! Exit status 1

[2020-11-12T12:08:15.362Z] npm ERR! 

[2020-11-12T12:08:15.362Z] npm ERR! Failed at the lisk-framework@0.7.0-alpha.3 test:integration script.

[2020-11-12T12:08:15.362Z] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

[2020-11-12T12:08:15.362Z] 

[2020-11-12T12:08:15.362Z] npm ERR! A complete log of this run can be found in:

[2020-11-12T12:08:15.362Z] npm ERR!     /home/lisk/.npm/_logs/2020-11-12T12_08_15_196Z-debug.log

script returned exit code 1

--- a/config/devnet/genesis_block.json
+++ b/config/devnet/genesis_block.json
@@ -2,11 +2,11 @@
 	"header": {
 		"generatorPublicKey": "",
 		"height": 0,
-		"id": "e266346995f4f848de415ace12be8d287bb58ac01baf819634a385ade26f4c11",
+		"id": "5fcd9e0023a6cde6b6add2dee9038eaa432b0306aca24d55028ba28a43dff21e",
 		"previousBlockID": "",
 		"reward": "0",
 		"signature": "",
-		"timestamp": 1601474827,
+		"timestamp": 1610643809,
 		"transactionRoot": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
 		"version": 0,
 		"asset": {

--- a/framework-plugins/lisk-framework-report-misbehavior-plugin/test/integration/cleanup_job.spec.ts
+++ b/framework-plugins/lisk-framework-report-misbehavior-plugin/test/integration/cleanup_job.spec.ts
@@ -13,6 +13,7 @@
  */
 import * as os from 'os';
 import * as path from 'path';
+import * as fs from 'fs-extra';
 import { getAddressAndPublicKeyFromPassphrase } from '@liskhq/lisk-cryptography';
 import {
 	blockHeaderAssetSchema,
@@ -63,13 +64,15 @@ describe('Clean up old blocks', () => {
 	);
 	const dbKey = 'the_db_key';
 
-	beforeEach(() => {
+	beforeEach(async () => {
 		reportMisbehaviorPlugin = new (ReportMisbehaviorPlugin as any)();
 		(reportMisbehaviorPlugin as any)._channel = channelMock;
+		const dataPath = path.join(os.homedir(), '.lisk/report-misbehavior-plugin/data/integration/db');
+		await fs.remove(dataPath);
 		(reportMisbehaviorPlugin as any).options = {
 			fee: '100000000',
 			clearBlockHeadersInterval: 1,
-			dataPath: path.join(os.homedir(), '~/.lisk/report-misbehavior-plugin/data/integration/db'),
+			dataPath,
 		};
 		reportMisbehaviorPlugin.schemas = {
 			block: blockSchema,

--- a/framework-plugins/lisk-framework-report-misbehavior-plugin/test/utils/application.ts
+++ b/framework-plugins/lisk-framework-report-misbehavior-plugin/test/utils/application.ts
@@ -80,7 +80,11 @@ export const getApplication = (
 		},
 	} as PartialApplicationConfig;
 
-	const app = Application.defaultApplication(genesisBlock, config);
+	const genesis = {
+		...genesisBlock,
+		header: { ...genesisBlock.header, timestamp: Math.floor(Date.now() / 1000) },
+	};
+	const app = Application.defaultApplication(genesis, config);
 	app.registerPlugin(ReportMisbehaviorPlugin, { loadAsChildProcess: false });
 	return app;
 };

